### ISSUE TITLE
Collab editor broken

### ISSUE THREAD

**Issue by bradleyayers** [ProseMirror/prosemirror#675]
There's a problem with the collab editor on https://prosemirror.net/examples/collab/#edit-Example

Given the document:

![image](https://user-images.githubusercontent.com/105820/28508012-c0aaed00-707a-11e7-9135-e8c214a5db52.png)

When pressing enter at the end of the line, an exception is thrown:

```
Uncaught TypeError: Cannot read property 'attrs' of null
    at canSplit (https://prosemirror.net/examples/prosemirror.js:98:1473)
    at https://prosemirror.net/examples/prosemirror.js:171:1554
    at Plugin.<anonymous> (https://prosemirror.net/examples/prosemirror.js:160:912)
    at https://prosemirror.net/examples/prosemirror.js:119:2397
    at EditorView.someProp (https://prosemirror.net/examples/prosemirror.js:180:5019)
    at readDOMChange (https://prosemirror.net/examples/prosemirror.js:119:2353)
    at DOMChange.finish (https://prosemirror.net/examples/prosemirror.js:119:6868)
    at editHandlers.input (https://prosemirror.net/examples/prosemirror.js:128:9630)
    at HTMLDivElement.e.dom.addEventListener.e.eventHandlers.(anonymous function) (https://prosemirror.net/examples/prosemirror.js:128:358)
```

**Comment by adrianheine** [comment]
I can reproduce the issue on the website, but not locally with the current `website` trunk. `npm ls --depth 0`:
```
prosemirror-website@0.0.1 /path
├── browserify@13.3.0
├── bubleify@0.7.0
├── builddocs@0.1.3
├── codemirror@5.28.0
├── crel@3.0.1
├── ecstatic@1.4.1
├── getdocs@0.6.0
├── glob@6.0.4
├── markdown-it@5.1.0
├── markdown-it-deflist@2.0.3
├── moduleserve@0.8.3
├── mold-template@2.0.1
├── prosemirror-collab@0.22.0
├── prosemirror-commands@0.22.0
├── prosemirror-example-setup@0.22.0
├── prosemirror-history@0.22.0
├── prosemirror-inputrules@0.22.0
├── prosemirror-keymap@0.22.1
├── prosemirror-markdown@0.22.0
├── prosemirror-menu@0.22.0
├── prosemirror-model@0.22.0
├── prosemirror-schema-basic@0.22.0
├── prosemirror-schema-list@0.22.0
├── prosemirror-schema-table@0.22.0
├── prosemirror-state@0.22.0
├── prosemirror-transform@0.22.2
├── prosemirror-view@0.22.0
├── punycode@1.4.1
├── uglifyify@3.0.4
└── watchify@3.9.0
```

**Comment by marijnh** [comment]
The website demo was still running with prosemirror-transform 0.22.0, and thus didn't have the fix for #667. I've upgraded.
